src/scripts/AddingReviewedMigration.js

```diff
@@ -0,0 +1,42 @@
+import { getConfig } from '../config/config.js';
+import { connectToDatabase } from '../api/db/connect.js';
+import { isImageReviewed } from '../api/db/models/utils.js';
+import Image from '../api/db/schemas/Image.js';
+
+async function updateImages() {
+    try {
+        const config = await getConfig();
+        const dbClient = await connectToDatabase(config);
+
+        let skip = 0;
+        let limit = 1; // how many images to fetch at a time
```

Comment:
Preliminary version of the script, keeping this at 1 for safety purposes until fully tested. Maybe do 1000 at a time

Reply:
Taking a look at `updateDocument.js` to see if this script is even needed